source/_cookbook/automation_kodi_dynamic_input_select.markdown

```diff
@@ -0,0 +1,156 @@
+---
+layout: page
+title: "Dynamic input select with Kodi media"
+description: "How to configure an `input_select` that can play in Kodi the latest entries added to the library"
+date: 2017-05-16 19:05
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: Automation Examples
+---
+
+This requires a [Kodi](/components/mediaplayer.kodi/) media player, and consists of commands that perform queries in the Kodi library, and a selection box (`input_select`) that shows the available options. By clicking the commands, the selection box is updated, and when selecting an option, Kodi plays the media.
+
+Option filling is done in an [AppDaemon](https://home-assistant.io/docs/ecosystem/appdaemon/tutorial/) app, which listens for events that are triggered with the results of the Kodi JSONRPC API queries when scripts run. This application also listens to the selection box and calls the play media service with the corresponding parameters.
+
+The media player yaml config:
+
+```yaml
+media_player:
+- platform: kodi
+  name: Kodi
+  host: 192.168.0.123
+```
+
+The `input_select` with the scripts and a group:
+
+```yaml
+input_select:
+  kodi_results:
+    name: Kodi API results
+    options:
+     - Nothing to do
+
+script:
+  get_recent_movies:
+    alias: Last added movies
+    sequence:
+      - service: media_player.kodi_call_method
+        data:
+          entity_id: media_player.kodi
+          method: VideoLibrary.GetRecentlyAddedMovies
+          properties:
+            - title
+            - year
+            - file
+            - lastplayed
+
+  get_recent_episodes:
+    alias: Last TV shows
+    sequence:
+      - service: media_player.kodi_call_method
+        data:
+          entity_id: media_player.kodi
+          method: VideoLibrary.GetRecentlyAddedEpisodes
+          properties:
+            - showtitle
+            - file
+            - lastplayed
+            - firstaired
+
+  get_pvr_channels:
+    alias: TV Channels
+    sequence:
+      - service: media_player.kodi_call_method
+        data:
+          entity_id: media_player.kodi
+          method: PVR.GetChannels
+          channelgroupid: 1
+
+group:
+  Media Select:
+    control: hidden
+    entities:
+      - input_select.kodi_results
+      - script.get_recent_movies
+      - script.get_recent_episodes
+      - script.get_pvr_channels
+```
+
+The AppDaemon app:
```

Comment:
Could you add in some more about the application and maybe some filler text about how to install it?
Not needed but considering this can be the first exposure to AppDaemon for the reader.